src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Mappers/Apprentice/RecognisePriorLearningViewModelToResultMapper.cs

```diff
@@ -79,4 +81,43 @@ public async Task<RecognisePriorLearningResult> Map(PriorLearningDetailsViewMode
             };
         }
     }
+
+    public class PriorLearningDataViewModelToResultMapper : IMapper<PriorLearningDataViewModel, RecognisePriorLearningResult>
+    {
+        private readonly IOuterApiService _outerApiService;
+        private readonly ICommitmentsApiClient _commitmentsApiClient;
+
+        public PriorLearningDataViewModelToResultMapper(IOuterApiService outerApiService, ICommitmentsApiClient commitmentsApiClient)
+        {
+            _outerApiService = outerApiService;
+            _commitmentsApiClient = commitmentsApiClient;
+        }
+
+        public async Task<RecognisePriorLearningResult> Map(PriorLearningDataViewModel source)
+        {
+            var update = _outerApiService.PriorLearningData(source.CohortId, source.DraftApprenticeshipId,
```

Comment:
It would be better if methods in this service were prefixed with a verb to indicate whether we're getting or posting data, etc. I assume this is an update.